src/lib/y2partitioner/actions/add_partition.rb

```diff
@@ -57,29 +63,77 @@ def size
 
     protected
 
+      # @return [Controllers::Partition]
       attr_reader :part_controller
 
       # @see TransactionWizard
       def sequence_hash
         {
-          "ws_start" => "type",
-          "type"     => { next: "size" },
-          "size"     => { next: new_blk_device_step1, finish: :finish }
+          "ws_start"          => "delete_filesystem",
```

Comment:
And what about disks directly used as PV or as part of a RAID or similar situations? Don't we want to handle them in a similar way?

Reply:
Good point. We should prevent modification of used devices.